src/main/java/com/assignment/Assignment11.java

```diff
@@ -0,0 +1,14 @@
+package com.assignment;
+
+import java.util.ArrayList;
+
+public class Assignment11 {
+    public ArrayList getSportExceptAtIndex(ArrayList sport, int index) {
+        ArrayList<String> newSportArray = new ArrayList<>();
+        for (int i = 0; i < sport.size(); i++)
+            if (i != index) {
+                newSportArray.add(String.valueOf(sport.get(i)));
```

Comment:
Why do we need to convert into string, when we already have a string? This should be simple as this :
`newSportArray.add(sport.get(i))`